### ISSUE TITLE
Weekend with Gwen

### ISSUE THREAD

**Issue by garrett-sutton** [garrett-sutton/garrett-sutton.github.io#70]
Grace went to a bachelorette weekend in RI for Memorial Day weekend 2024. I stayed home with Gwen.

**Comment by garrett-sutton** [comment]
Friday night 
- daycare pick-up 
- nothing too special

**Comment by garrett-sutton** [comment]
Saturday 
- early wake up ~5:30
- long long morning nap (2.5 hours)
- eat time #2 and then head out
- ran 50 minutes on Badger trail and then walked to Barique for coffee/lunch
- home to eat again, play and eventually another nap
- had a lot of fun playing hide/seek and tickle as I raced around her bouncer
- 4oz of formula went good
- a bit of a push to keep her awake to better space her food. We went for a late walk around 4:30 that got her to sleep a bit. I wanted her to sleep but I also was trying to be conscious of not stacking feeding too close 
- also poop. And poop up the back.

**Comment by garrett-sutton** [comment]
Sunday
- slept in until ~6:30
- church at 8:30. Very awake throughout and pleasant too
- slept on way home, ate and then went back to sleep for nearly 3 hours
- pouring rain all day but thought we had a little window for a walk in late afternoon. Unfortunately got caught in some rain as we were getting back 
- chat with my parents for a while. Gwen played very nice. Bouncer, ate and then floor.
   - 6 oz of formula went super smooth
   - had to hang up because of another poop up the back 
- talked with Grace on the phone for a bit 
- I made dinner and Gwen snacked on some sweet potato bar. Could tell she was getting a bit upset
- made our way to her room and she rolled all over the place as I crawled around her. She was super super happy

**Comment by garrett-sutton** [comment]
monday
- slept until ~5:30
- played in the playpen after eating 
- fell asleep in the playpen
   - woke up and continued just rolling around for a while
- ate around 9:30.
- Immediately after eating, went out to run, burpees, target
- home by 12:30 and she fell asleep on the floor as I was prepping bottle
- Woke her up to eat and immediately put her down. She slept for 3 hours.
- After she woke up and ate, I moved her around with me as I was finishing up some chores
- Then we went for a quick walk, came home for Gwen to watch me reorganize the storage bin
- bath and  a bit of play before Grace got home at 7

**Comment by garrett-sutton** [comment]
My feelings
- I was stressed about this weekend From the beginning. I've spent time at home alone with Gwen but never for even a whole day/night. And admittedly, I know that I/we are pretty dependent on Grace. Worry around the little things, like when would I go to the bathroom shower or eat to the bigger things like what's if she's up all night and I can't console her or how was it going to work for food all lingering in my head.
- These fears continued up until and even throughout the weekend, but as the weekend went on my confidence grew and it felt more and more manageable and even fun. It was a huge relief/step that we were able to go run with basically no issue 
- we definitely had our moments throughout the weekend. Largely with just her needing me to be close by and me needing to get something else done whether that be for her or otherwise. But at the end of the day, it was ok if she was upset for a bit as I was doing a thing. 
- gained a bunch of confidence with Gwen this weekend
- Felt like she grew over the course of the weekend and that was a lot of fun for me.
- with her being where she is developmentally, I feel like I got to know her better
- and this really ended up being a lot of fun
